refarch-gateway/src/main/java/de/muenchen/refarch/gateway/ApiGatewayApplication.java

```diff
@@ -30,7 +31,7 @@
  * "https://cloud.spring.io/spring-cloud-gateway/reference/html/">https://cloud.spring.io/spring-cloud-gateway/reference/html/</a>
  */
 @SpringBootApplication
-@EnableConfigurationProperties(SecurityProperties.class)
+@EnableConfigurationProperties({ SecurityProperties.class, HazelcastProperties.class })
```

Comment:
Why are all Properties classes listed here separately. Shouldn't it be possible to make use of Spring automatically scanning the base package? e.g. via `ConfigurationPropertiesScan`?

Reply:
Done